Loop/Managers/DeviceDataManager.swift

```diff
@@ -391,10 +410,58 @@ final class DeviceDataManager: CarbStoreDelegate, CarbStoreSyncDelegate, DoseSto
                     nsPumpStatus = nil
                 }
                 self.nightscoutDataManager.uploadDeviceStatus(nsPumpStatus)
+                completion?()
             }
         }
     }
 
+    private func updatePumpCGMSensorStatus(_ events: [TimestampedGlucoseEvent]) -> Void {
```

Comment:
And wrt function and variable names, I agree with renaming 'pump cgm data' to 'enlite data'